app/models/runs_extractors.rb

```diff
@@ -0,0 +1,75 @@
+class RunsExtractors
+  attr_reader :reducible, :extractors
```

Comment:
This reducible attr probably isn't necessary and doesn't make sense on Runs _Extractor_, which isn't concerned with reducibles and gets its workflow via id in the classification.

Reply:
Fair point. In that same line of thinking, the `extractors_runner` method doesn't belong in ClassificationPipeline either, since it shouldn't be mixed into Project or other Reducibles. And then Pipeline might as well go away, and the remaining methods moved to Reducible